cell_workers/pokemon_catch_worker.py

```diff
@@ -105,6 +106,19 @@ def work(self):
                                 if status is 3:
                                     print('[x] Oh no! ' + str(pokemon_name) + ' vanished! :(')
                                 if status is 1:
+                                    self.api.get_inventory()
```

Comment:
@mohitmun leave this for another pull request, because that thing is big and dangerous.